test/e2e/ingress_utils.go

```diff
@@ -616,17 +616,19 @@ func gcloudList(resource, regex, project string, out interface{}) {
 	// so we only look at stdout.
 	command := []string{
 		"compute", resource, "list",
-		fmt.Sprintf("--regex=%v", regex),
+		fmt.Sprintf("--regexp=%v", regex),
 		fmt.Sprintf("--project=%v", project),
 		"-q", "--format=json",
 	}
 	output, err := exec.Command("gcloud", command...).Output()
 	if err != nil {
 		errCode := -1
+		errMsg := ""
 		if exitErr, ok := err.(utilexec.ExitError); ok {
 			errCode = exitErr.ExitStatus()
+			errMsg = exitErr.Error()
```

Comment:
I don't think this includes stderr, which is a [separate field in `ExitError`](https://golang.org/pkg/os/exec/#ExitError)